.github/workflows/conformance-eks.yaml

```
@@ -112,7 +112,7 @@ concurrency:
 
 env:
   test_concurrency: ${{ inputs.test-concurrency || 3 }}
-  clusterName: ${{ github.repository_owner }}-${{ github.event.repository.name }}-${{ github.run_id }}-${{ inputs.UID }}-${{ github.run_attempt }}
+  clusterNamePrefix: ${{ github.repository_owner }}-${{ github.event.repository.name }}-${{ github.run_id }}-${{ inputs.UID }}-${{ github.run_attempt }}
```

```suggestion
  clusterNamePrefix: ${{ github.repository_owner }}-${{ github.event.repository.name }}-${{ github.run_id }}
```
same as above, regarding the run_attempt value, which increases/changes on every retry.

However, I would also suggest not using inputs.UID in this workflow and not in the aws-cni one?
```
UID:
        description: "A second number, to differentiate multiple runs of the same workflow by the same PR"
        required: false
        default: 1
```

I don't think this adds any value for when it would become useful:
1. k8s-matrix.yaml pointing 2+ clusters to the same region - they would both have "1" as value
2. retries - clusters in the same region would still both have "1"

I see UID is part of the concurrency group - in that case, it would make sense - but I don't think it would work as expected if the input parameter is not passed and 2 clusters run on the same region.

---

The UID is being used in workflows that call this workflow multiple times, for example, conformance-kpr.yaml so that we can distinguish multiple runs.